protocol/ledger.re

```diff
@@ -1,86 +1,42 @@
 open Helpers;
-open Exn_noop;
 
 module Wallet_map = Map_with_yojson_make(Wallet);
 [@deriving yojson]
-type t = {
-  free: Wallet_map.t(Amount.t),
-  frozen: Wallet_map.t(Amount.t),
-};
-
-let empty = {free: Wallet_map.empty, frozen: Wallet_map.empty};
+type t = {free: Wallet_map.t(Amount.t)};
```

Comment:
This is still a record because in a future PR it will be extended again to support withdraws